src/renderer/families/cosmos/operationDetails.js

```diff
@@ -0,0 +1,375 @@
+  /** $FlowFixMe */
+  const url = getAddressExplorer(getDefaultExplorerView(currency), address);
+  if (url) openURL(url);
+};
+
+/** @TODO move this in common */
+const formatDelegationsInfo = (
+  delegations: CosmosDelegationInfo[],
+  validators: CosmosValidatorItem[],
+): {
+  validator: ?CosmosValidatorItem,
+  address: string,
+  amount: BigNumber,
+}[] => {
+  return delegations.map((d, i, arr) => ({
+    validator: validators.find(v => v.validatorAddress === d.address),
+    address: d.address,
+    amount: d.amount,
+  }));
+};
+
+type OperationDetailsDelegationProps = {
+  discreet: boolean,
+  unit: Unit,
+  currency: Currency,
+  delegations: Array<CosmosDelegationInfo>,
+  account: Account,
+  isTransactionField?: boolean,
+  cosmosValidators: CosmosValidatorItem[],
+};
+
+export const OperationDetailsDelegation = ({
+  discreet,
+  unit,
+  currency,
+  delegations,
+  account,
+  isTransactionField,
+  cosmosValidators,
+}: OperationDetailsDelegationProps) => {
+  const formattedDelegations = formatDelegationsInfo(delegations, cosmosValidators);
+
+  return (
+    <Box>
+      {!isTransactionField && (
+        <OpDetailsTitle>
+          <Trans i18nKey={"operationDetails.extra.validators"} />
+        </OpDetailsTitle>
+      )}
+
+      {formattedDelegations
+        .map(({ amount, ...delegation }) => ({
+          ...delegation,
+          amount: formatCurrencyUnit(unit, BigNumber(amount), {
+            disableRounding: false,
+            alwaysShowSign: false,
+            showCode: true,
+            discreet,
+          }),
+        }))
+        .map(({ amount, address, validator }, i) => (
+          <OpDetailsData key={address + i}>
+            <OpDetailsVoteData>
+              <Box>
+                <Text>
+                  <Trans
+                    i18nKey="operationDetails.extra.votesAddress"
+                    values={{
+                      votes: amount,
+                      name: validator ? validator.name : address,
+                    }}
+                  >
+                    <Text ff="Inter|SemiBold">{""}</Text>
+                    {""}
+                    <Text ff="Inter|SemiBold">{""}</Text>
+                  </Trans>
+                </Text>
+              </Box>
+              <Address onClick={redirectAddress(currency, address)}>{address}</Address>
+            </OpDetailsVoteData>
+          </OpDetailsData>
+        ))}
+    </Box>
+  );
+};
+
+type OperationDetailsExtraProps = {
+  extra: { [key: string]: any },
+  type: string,
+  account: Account,
+};
+
+const OperationDetailsExtra = ({ extra, type, account }: OperationDetailsExtraProps) => {
+  const unit = getAccountUnit(account);
+  const currency = getAccountCurrency(account);
+  const discreet = useDiscreetMode();
+  const { validators: cosmosValidators } = useCosmosPreloadData();
+
+  switch (type) {
+    case "DELEGATE": {
+      const { validators: delegations } = extra;
```

Comment:
the fact there is `validators` in two different part of our data makes me thing we don't have found the good wording yet. I would prefer we find a better non colliding wording and that we don't have to rename every time.
cc @henri-ly